protocol/x/clob/keeper/clob_pair.go

```diff
@@ -373,6 +373,13 @@ func (k Keeper) validateOrderAgainstClobPairStatus(
 				clobPair.Status,
 			)
 		}
+	case types.ClobPair_STATUS_FINAL_SETTLEMENT:
```

Comment:
Somewhat unrelated, but why is it a valid state transition for a final settlement CLOB pair to transition into `INITIALIZING`? [source](https://github.com/dydxprotocol/v4-chain/blob/0f0d706c80054911482720426bd17caf11294c97/protocol/x/clob/types/clob_pair.go#L20-L22)

Reply:
answer here: https://github.com/dydxprotocol/v4-chain/pull/829#discussion_r1430742315